internal/service/streamprocessor/resource_test.go

```diff
@@ -61,6 +68,25 @@ func TestAccStreamProcessor_basic(t *testing.T) {
 				ImportStateVerify:       true,
 				ImportStateVerifyIgnore: []string{"stats"},
 			},
+		}}
+}
+
+func TestAccStreamProcessor_JSONWhiteSpaceFormat(t *testing.T) {
+	var (
+		projectID                  = acc.ProjectIDExecution(t)
+		processorName              = "new-processor-json-unchanged"
+		instanceName               = acc.RandomName()
+		sampleSrcConfigExtraSpaces = connectionConfig{connectionType: connTypeSample, pipelineStepIsSource: true, extraWhitespace: true}
+	)
+	resource.ParallelTest(t, resource.TestCase{
+		ProtoV6ProviderFactories: acc.TestAccProviderV6Factories,
+		PreCheck:                 func() { acc.PreCheckBasic(t) },
+		CheckDestroy:             checkDestroyStreamProcessor,
+		Steps: []resource.TestStep{
+			{
+				Config: config(t, projectID, instanceName, processorName, streamprocessor.CreatedState, sampleSrcConfigExtraSpaces, testLogDestConfig),
+				Check:  composeStreamProcessorChecks(projectID, instanceName, processorName, streamprocessor.CreatedState, false, false),
+			},
```

Comment:
I don't see the point of testing that a json string with whitespaces is valid. I think this is not provide any value in the test suite and can be distracting

Reply:
Please see [comment below for more explanation](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2519#discussion_r1727034943)

Reply:
ok to resolve @oarbusi ?